pkg/lang/frontend/starlark/universe/const.go

```diff
@@ -19,4 +19,5 @@ const (
 	ruleShell     = "shell"
 	ruleRun       = "run"
 	ruleGitConfig = "git_config"
+	ruleExpose    = "expose"
```

Comment:
* https://github.com/tensorchord/envd/pull/769#pullrequestreview-1072455645
* Will it be better to place `expose` under `runtime`?

Reply:
LGTM